kbs/src/token/coco.rs

```diff
@@ -21,7 +21,7 @@ pub struct CoCoAttestationTokenVerifier {
 }
 
 impl CoCoAttestationTokenVerifier {
-    pub fn new(config: &AttestationTokenVerifierConfig) -> Result<Self> {
+    pub async fn new(config: &AttestationTokenVerifierConfig) -> Result<Self> {
```

Comment:
We do not need this if we want outer called to be async.

Reply:
ah ok, let me drop

Reply:
done